### ISSUE TITLE
Swipe from messages view back

### ISSUE THREAD

**Issue by Wikunia** [zulip/zulip-mobile#4036]
I'm kind of sure this issue exists here somewhere because I find it kinda obvious but I wasn't able to find it. 
When viewing messages one has to click the back button at the top left to go back. It would be nice to also have swipe from left to right to have this feature. Currently swiping is not in use on that screen and it seems intuitive for people who used slack before.

**Comment by chrisbobbe** [comment]
Are you on Android or iOS? I believe this is intentionally included on iOS, and it's working for me; maybe you're on Android, or you've found a place on iOS where it doesn't work?

**Comment by Wikunia** [comment]
I'm on Android 10.

**Comment by chrisbobbe** [comment]
> Thought this repository is for both systems as it's a react native app, isn't it?

Yes and yes, but in some areas, each platform has its own recommendations and requirements for how to do things, which means some differences in implementation, and React Native lets you do this fairly easily. Navigation has been one of these areas.

— and, it looks like Android 10 does indeed support and recommend gesture-based back navigation! ([doc](https://developer.android.com/training/gestures/gesturenav))

**Comment by Aman-Maheshwari** [comment]
@Wikunia @chrisbobbe  is this issue still active ? can i work on this for android ?

**Comment by Wikunia** [comment]
Great @Aman-Maheshwari I'm looking forward to it!

**Comment by Aman-Maheshwari** [comment]
@Wikunia @chrisbobbe  Please check the PR. Is it working fine for the issue ?
